### PR TITLE
Add flyway blog

### PR DESCRIPTION
**Pull request by adela-bytebase**

### CODE REVIEW

File: content/blog/bytebase-vs-flyway.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+title: 'Bytebase vs. Flyway: a side-by-side database DevOps tool comparison for 2023'
```
Comment (tianzhou) [inline content/blog/bytebase-vs-flyway.md:2]: database CI/CD tool. Be consistent with https://www.bytebase.com/blog/bytebase-vs-liquibase/

**Review by tianzhou — CHANGES_REQUESTED** [review]
Please remove other unrelated changes.

File: src/app/docs/[...slug]/page.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ export default function DocPage({ params }: { params: { slug: string[] } }) {
         </PostLayout>
       </article>
       {tableOfContents && tableOfContents.length > 0 && (
-        <div className="col-span-3 col-end-13 ml-auto w-full max-w-[314px] pt-2.5 pb-28 xl:max-w-none lg:hidden">
+        <div className="col-span-3 col-end-13 ml-auto w-full max-w-[314px] pb-28 pt-2.5 xl:max-w-none lg:hidden">
```
Comment (adela-bytebase) [inline src/app/docs/[...slug]/page.tsx:70]:
```suggestion
        <div className="col-span-3 col-end-13 ml-auto w-full max-w-[314px] pt-2.5 pb-28 xl:max-w-none lg:hidden">
```

File: src/app/docs/layout.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ export default function DocLayout({ children }: { children: React.ReactNode }) {
       <div className="container grid grid-cols-12 gap-x-10 pt-[136px] 2xl:pt-[140px] xl:gap-x-9 xl:pt-32 lg:gap-x-6 md:mt-6 md:gap-x-5 md:pt-0 sm:gap-x-4">
         <div
           className="sticky top-10 col-span-3 h-[calc(100vh-40px)] overflow-hidden 
-        before:absolute before:top-0 before:left-0 before:z-10 before:h-[72px] before:w-full before:bg-docs-sidebar 
+        before:absolute before:left-0 before:top-0 before:z-10 before:h-[72px] before:w-full before:bg-docs-sidebar
```
Comment (adela-bytebase) [inline src/app/docs/layout.tsx:22]:
```suggestion
        before:absolute before:top-0 before:left-0 before:z-10 before:h-[72px] before:w-full before:bg-docs-sidebar 
```

File: src/components/pages/contact/logos/logos.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ const Logos = () => {
   return (
     <div className="col-span-5 mt-9 border-t border-gray-90 pt-9 lg:col-span-full lg:mt-0 lg:border-none lg:pt-0">
       <p className="text-18 leading-extra-tight text-gray-40">Trusted by engineers worldwide</p>
-      <ul className="it mt-7 flex flex-wrap items-start gap-y-5 gap-x-9 2xl:gap-x-7 xl:gap-x-9 md:gap-x-6 xs:gap-x-5 xs:gap-y-6">
+      <ul className="it mt-7 flex flex-wrap items-start gap-x-9 gap-y-5 2xl:gap-x-7 xl:gap-x-9 md:gap-x-6 xs:gap-x-5 xs:gap-y-6">
```
Comment (adela-bytebase) [inline src/components/pages/contact/logos/logos.tsx:7]:
```suggestion
      <ul className="it mt-7 flex flex-wrap items-start gap-x-9 gap-y-5 2xl:gap-x-7 xl:gap-x-9 md:gap-x-6 xs:gap-x-5 xs:gap-y-6">
```

File: src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx [hunk 5]
```diff
@@ -76,21 +76,21 @@ const MobileSidebar = ({
     <LazyMotion features={domAnimation}>
       <nav className={clsx('mobile-sidebar pt-[75px]', className)} ref={wrapperRef}>
         <button
-          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-t border-b border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
+          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-b border-t border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
```
Comment (adela-bytebase) [inline src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx:79]:
```suggestion
          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-t border-b border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
```

File: src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx [hunk 6]
```diff
@@ -76,21 +76,21 @@ const MobileSidebar = ({
...
           type="button"
           onClick={toggleMenu}
         >
           <span>Dictionary menu</span>
           <span className="relative">
             <span
               className={clsx(
-                'absolute top-1/2 -left-3 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
+                'absolute -left-3 top-1/2 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```
Comment (adela-bytebase) [inline src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx:87]:
```suggestion
                'absolute -left-3 top-1/2 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```

File: src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx [hunk 7]
```diff
@@ -76,21 +76,21 @@ const MobileSidebar = ({
...
                 isOpen ? 'rotate-45' : 'rotate-[135deg]',
               )}
             />
             <span
               className={clsx(
-                'absolute top-1/2 -left-[7px] h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
+                'absolute -left-[7px] top-1/2 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```
Comment (adela-bytebase) [inline src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx:93]:
```suggestion
                'absolute top-1/2 -left-[7px] h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```

File: src/components/pages/docs/post-layout/post-layout.tsx [hunk 8]
```diff
@@ -37,7 +37,7 @@ const PostLayout = ({
       </h1>
       {featureImage && (
         <Image
-          className="my-11 w-full rounded lg:mt-10 lg:mb-8 sm:mt-5 sm:mb-6"
+          className="my-11 w-full rounded lg:mb-8 lg:mt-10 sm:mb-6 sm:mt-5"
```
Comment (adela-bytebase) [inline src/components/pages/docs/post-layout/post-layout.tsx:40]:
```suggestion
          className="my-11 w-full rounded lg:mt-10 lg:mb-8 sm:mt-5 sm:mb-6"
```

File: src/styles/typography.css [hunk 9]
```diff
@@ -125,7 +125,7 @@
 
     & h2,
     & h3 {
-      @apply mt-9 mb-4 text-24 leading-tight lg:mt-6 lg:mb-3 lg:text-20 md:mt-5 md:mb-2 md:text-18;
+      @apply mb-4 mt-9 text-24 leading-tight lg:mb-3 lg:mt-6 lg:text-20 md:mb-2 md:mt-5 md:text-18;
```
Comment (adela-bytebase) [inline src/styles/typography.css:128]:
```suggestion
      @apply mt-9 mb-4 text-24 leading-tight lg:mt-6 lg:mb-3 lg:text-20 md:mt-5 md:mb-2 md:text-18;
```

File: src/styles/typography.css [hunk 10]
```diff
@@ -16,7 +16,7 @@
 
     @apply leading-normal xl:text-16 md:leading-snug;
 
-    @apply prose-headings:mt-11 prose-headings:mb-5 prose-headings:font-semibold prose-headings:text-gray-15 prose-p:my-5 prose-p:text-gray-15 prose-a:break-words prose-a:font-semibold prose-a:text-primary-1 prose-a:no-underline prose-a:transition-colors prose-a:duration-200 hover:prose-a:text-gray-60 prose-ul:my-5 prose-li:text-gray-15 xl:prose-headings:mt-10 xl:prose-headings:mb-4 xl:prose-p:my-4 md:prose-headings:mt-9 sm:prose-headings:mt-6 sm:prose-headings:leading-tight sm:prose-p:my-3;
+    @apply prose-headings:mb-5 prose-headings:mt-11 prose-headings:font-semibold prose-headings:text-gray-15 prose-p:my-5 prose-p:text-gray-15 prose-a:break-words prose-a:font-semibold prose-a:text-primary-1 prose-a:no-underline prose-a:transition-colors prose-a:duration-200 hover:prose-a:text-gray-60 prose-ul:my-5 prose-li:text-gray-15 xl:prose-headings:mb-4 xl:prose-headings:mt-10 xl:prose-p:my-4 md:prose-headings:mt-9 sm:prose-headings:mt-6 sm:prose-headings:leading-tight sm:prose-p:my-3;
```
Comment (adela-bytebase) [inline src/styles/typography.css:19]:
```suggestion
    @apply prose-headings:mt-11 prose-headings:mb-5 prose-headings:font-semibold prose-headings:text-gray-15 prose-p:my-5 prose-p:text-gray-15 prose-a:break-words prose-a:font-semibold prose-a:text-primary-1 prose-a:no-underline prose-a:transition-colors prose-a:duration-200 hover:prose-a:text-gray-60 prose-ul:my-5 prose-li:text-gray-15 xl:prose-headings:mt-10 xl:prose-headings:mb-4 xl:prose-p:my-4 md:prose-headings:mt-9 sm:prose-headings:mt-6 sm:prose-headings:leading-tight sm:prose-p:my-3;
```

File: src/components/pages/sql-review-guide/mobile-sidebar/mobile-sidebar.tsx [hunk 11]
```diff
@@ -92,21 +92,21 @@ const MobileSidebar = ({
...
+          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-b border-t border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
           type="button"
           onClick={toggleMenu}
         >
           <span>Rules menu</span>
           <span className="relative">
             <span
               className={clsx(
-                'absolute top-1/2 -left-3 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
+                'absolute -left-3 top-1/2 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```
Comment (adela-bytebase) [inline src/components/pages/sql-review-guide/mobile-sidebar/mobile-sidebar.tsx:103]:
```suggestion
                'absolute top-1/2 -left-3 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```

File: src/components/pages/sql-review-guide/mobile-sidebar/mobile-sidebar.tsx [hunk 12]
```diff
@@ -92,21 +92,21 @@ const MobileSidebar = ({
     <LazyMotion features={domAnimation}>
       <nav className={clsx('mobile-sidebar pt-[75px]', className)} ref={wrapperRef}>
         <button
-          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-t border-b border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
+          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-b border-t border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
```
Comment (adela-bytebase) [inline src/components/pages/sql-review-guide/mobile-sidebar/mobile-sidebar.tsx:95]:
```suggestion
          className="flex w-full cursor-pointer appearance-none items-center justify-between text-ellipsis border-t border-b border-gray-94 bg-gray-97 px-11 py-4 leading-none tracking-tight text-gray-15 outline-none transition-colors duration-200 hover:bg-gray-90 active:bg-gray-90 md:px-7 sm:px-4"
```

File: src/components/pages/home/logos/logos.tsx [hunk 13]
```diff
@@ -9,7 +9,7 @@ const Logos = () => {
         <h2 className="max-w-3xl text-center font-title text-72 font-semibold leading-none xl:max-w-2xl xl:text-56 md:max-w-lg md:text-48 sm:text-40">
           Trusted by engineers worldwide
         </h2>
-        <ul className="mt-16 grid max-w-[1133px] grid-cols-[repeat(4,auto)] flex-wrap justify-center gap-x-24 gap-y-12 2xl:mt-14 xl:mt-12 xl:max-w-[740px] xl:gap-y-8 xl:gap-x-16 md:mt-10 md:max-w-[580px] md:gap-y-7 md:gap-x-10 sm:mt-8 xs:flex xs:max-w-[460px] xs:flex-wrap xs:justify-center xs:gap-x-5 xs:gap-y-6">
+        <ul className="mt-16 grid max-w-[1133px] grid-cols-[repeat(4,auto)] flex-wrap justify-center gap-x-24 gap-y-12 2xl:mt-14 xl:mt-12 xl:max-w-[740px] xl:gap-x-16 xl:gap-y-8 md:mt-10 md:max-w-[580px] md:gap-x-10 md:gap-y-7 sm:mt-8 xs:flex xs:max-w-[460px] xs:flex-wrap xs:justify-center xs:gap-x-5 xs:gap-y-6">
```
Comment (adela-bytebase) [inline src/components/pages/home/logos/logos.tsx:12]:
```suggestion
        <ul className="mt-16 grid max-w-[1133px] grid-cols-[repeat(4,auto)] flex-wrap justify-center gap-x-24 gap-y-12 2xl:mt-14 xl:mt-12 xl:max-w-[740px] xl:gap-y-8 xl:gap-x-16 md:mt-10 md:max-w-[580px] md:gap-y-7 md:gap-x-10 sm:mt-8 xs:flex xs:max-w-[460px] xs:flex-wrap xs:justify-center xs:gap-x-5 xs:gap-y-6">
```

File: src/components/pages/docs/sidebar/sidebar.tsx [hunk 14]
```diff
@@ -14,7 +14,7 @@ const Sidebar = ({ data, expandedList }: SidebarProps) => {
   return (
     <aside className="sidebar">
       <AlgoliaSearch className="relative z-10" />
-      <nav className="scrollbar-hidden max-h-[calc(100vh-80px)] overflow-y-scroll pt-6 pb-10">
+      <nav className="scrollbar-hidden max-h-[calc(100vh-80px)] overflow-y-scroll pb-10 pt-6">
```
Comment (adela-bytebase) [inline src/components/pages/docs/sidebar/sidebar.tsx:17]:
```suggestion
      <nav className="scrollbar-hidden max-h-[calc(100vh-80px)] overflow-y-scroll pt-6 pb-10">
```

File: src/components/pages/contact/logos/logos.tsx (hunk 4 above)
Comment (adela-bytebase) [inline src/components/pages/contact/logos/logos.tsx:7]:
```suggestion
      <ul className="it mt-7 flex flex-wrap items-start gap-y-5 gap-x-9 2xl:gap-x-7 xl:gap-x-9 md:gap-x-6 xs:gap-x-5 xs:gap-y-6">
```

File: src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx [hunk 15]
```diff
@@ -84,7 +84,7 @@ const MobileSidebar = ({
           <span className="relative">
             <span
               className={clsx(
-                'absolute top-1/2 -left-3 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
+                'absolute -left-3 top-1/2 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```
Comment (adela-bytebase) [inline src/components/pages/database-glossary/mobile-sidebar/mobile-sidebar.tsx:87]:
```suggestion
                'absolute top-1/2 -left-3 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```

File: src/components/pages/sql-review-guide/mobile-sidebar/mobile-sidebar.tsx [hunk 16]
```diff
@@ -106,7 +106,7 @@ const MobileSidebar = ({
             />
             <span
               className={clsx(
-                'absolute top-1/2 -left-[7px] h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
+                'absolute -left-[7px] top-1/2 h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```
Comment (adela-bytebase) [inline src/components/pages/sql-review-guide/mobile-sidebar/mobile-sidebar.tsx:109]:
```suggestion
                'absolute top-1/2 -left-[7px] h-2 w-[1.5px] -translate-y-1/2 bg-current transition-transform duration-200',
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**